### ISSUE TITLE
Please introduce modern windows 11 insipired  UI please

### ISSUE THREAD

**Issue by fisforfaheem** [truong2d/mm#9]
many people want this app also and love it but it seems to outdated...jus tput one week and update to new Ui it's easier

**Comment by NeoTechni** [comment]
Please no, the functionality is more important than it's appearance. Don't waste time on this better spent fixing issues

**Comment by dan400man** [comment]
Hell no.  The UI is set it and forget it.  I would be disappointed if the author spent any time on this request.

**Comment by fisforfaheem** [comment]
Hell yes, It's just a suggestion for author to explore building the same
app using latest visual studio 2022, Which would automatically add new
features and stability without any hardwork.

I urge the author to update it's development tools and IDE to latest
available version and build the app on latest version



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
12/11/21,
03:05:53 PM

**Comment by Zeratoxx** [comment]
Also: Win11 UI is really annoying in my opinion. Especially the rounded corners. A hardware display has 90° angled corners so why not the windows also..? 

I was perfectly fine with the Win10 design, a bit more consequent implementation throughout all of the Windows components and a bit more optimization in terms of intuitive user experience (looking at old MS Store design) and all would have been fine in my eyes.

**Comment by fisforfaheem** [comment]
I agree with you. But Windows 10 was old really old in terms of so many
things and deserves some changes..as well as apps... FOCUS on new apps and
not windows 11



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
12/14/21,
04:20:55 PM
